ShopAPI/ShopAPI/Models/Product.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace ShopAPI.Models
+﻿using Microsoft.EntityFrameworkCore.Metadata.Internal;
```

Comment:
remove this using statement, as its not needed

Reply:
Done